Dockerfile

```diff
@@ -17,6 +17,7 @@ FROM alpine:${ALPINE_VERSION}
 
 # install python, git, bash
 RUN apk add --no-cache git git-lfs python3 bash
+RUN apk add docker
```

Comment:
This adds an extra Docker layer and alsos leave some apk cache files in the image. Is there a reason it can't be included in the previous line?

Reply:
Hi @manics great suggestion.  Made the change